changelogs/CHANGELOG.rst

```diff
@@ -5,10 +5,13 @@ Ansible Network Collection Release Notes
 .. contents:: Topics
 
 
-v1.0.0
+v1.0.1
 ======
 
 Release Summary
 ---------------
 
-- Released 1.0.0 with updated changelog.
+- Rereleased 1.0.0 with updated README.md and regenerated documentation.
```

Comment:
Just "README documentation update" should be enough.  "Rerelease" also sounds like we are overwriting 1.0.0.

Reply:
sounds good. Updated !